lua/wikis/commons/Condition.lua

```diff
@@ -76,27 +98,20 @@ local ConditionNode = Class.new(_ConditionNode,
 
 ---@return string
 function ConditionNode:toString()
-	return String.interpolate(
-		'[[${name}${comparator}${value}]]',
-		{
-			name = self.name:toString(),
-			comparator = self.comparator,
-			value = self.value
-		}
-	)
-end
+	local conditions = Array.map(self.comparator, function(comp)
```

Comment:
Handling for empty array input?

Reply:
Not nil but an empty array. Output would be `()` then

Reply:
```lua
	assert(#self.comparator > 0,'Invalid comparator for LPDB query condition')
```
This would be enough?

Reply:
agreed